src/RA_Dlg_MemBookmark.cpp

```diff
@@ -827,42 +826,50 @@ std::string Dlg_MemBookmark::ImportDialog()
 {
 	std::string str;
 
+	// With how repetive this is, it should be it's own function
 	if ( g_pCurrentGameData->GetGameID() == 0 )
 	{
 		MessageBox( nullptr, _T("ROM not loaded: please load a ROM first!"), _T("Error!"), MB_OK );
 		return str;
-	}
+	} // end if
 
-	IFileOpenDialog* pDlg = nullptr;
-	HRESULT hr = CoCreateInstance( CLSID_FileOpenDialog, NULL, CLSCTX_ALL, IID_IFileOpenDialog, reinterpret_cast<void**>( &pDlg ) );
-	if ( hr == S_OK )
+	// Never noticed this, nearly identical to the on in RA_Core
+	CComPtr<IFileOpenDialog> pDlg;
+
+	// Keep the scope as local as possible
+	
+	if (auto hr = CoCreateInstance(CLSID_FileOpenDialog, nullptr, CLSCTX_ALL, IID_IFileOpenDialog, 
+		reinterpret_cast<void**>(&pDlg)); SUCCEEDED(hr))
```

Comment:
these changes should be part of #28, not here.

Reply:
It would be preferable. The changes fit that PR, not this one. If for any reason either of these PR has to be reverted, it keeps the related changes separate.

Reply:
Well, it would be kinda funky. I could just copy and paste the original here and copy and paste what's here in #28.

Reply:
OK the original is mostly the way it was.